### PR TITLE
Add the `$idPrefix` config option to FormHelper

### LINKED ISSUE #6134: Set default idPrefix for Form Helper
Since the default idPrefix for the FormHelper is "null", this can lead to id collisions on a website.

As example:
I got an DB with the columns id, username, title, content, so the FormHelper creates all inputs by default with:
`<label for="content">...<//label><textarea name="content" id="content"></textarea>`

Since content is a common id for the content wrapper etc and those collisions might come up also with other ids, I would say it makes sense to set the default idPrefix for the FormHelper to "cakeForm" to reach relative unique ids like "cakeForm-title", "cakeForm-content" etc to avoid collisions at all.

The ID Prefix above was just an example, you get the idea.

### PR DESCRIPTION
**Pull request by jadb**
As it stands, even creating an `AppFormHelper` to define a custom `$_idPrefix` to be used fails because it is always overwritten by the `$options['idPrefix']` (even if that's `NULL`).

This PR fixes that edge case and adds the option to define a custom prefix using the config options, instead of forcing one to overwrite the `FormHelper`.

Closes #6134.

Update: I still need to write tests for this, but first wanted to see what fails as I just made the changes via web :)

### CODE REVIEW

File: src/View/Helper/FormHelper.php [hunk 1]
```diff
@@ -352,7 +354,9 @@ public function create($model = null, array $options = [])
             'idPrefix' => null,
         ];
 
-        $this->_idPrefix = $options['idPrefix'];
+        if (!is_null($options['idPrefix'])) {
```
Comment (dereuromark) [inline src/View/Helper/FormHelper.php]: !== null :)

Comment (jadb) [inline src/View/Helper/FormHelper.php]: Will change :)

**Comment by markstory** [comment]
Seems reasonable to me.

**Comment by lorenzo** [comment]
Good job :clap:
